build.gradle

```diff
@@ -171,6 +211,7 @@ allprojects {
     def betterStringsCoords = [ group: 'com.antkorwin', name: 'better-strings', version: betterStringsVersion ]
     annotationProcessor(betterStringsCoords)
     testAnnotationProcessor(betterStringsCoords)
+    implementation 'org.jsoup:jsoup:1.15.1'
```

Comment:
In fact, it's needed only on **classpath of Gradle itself** (rather than classpath of any compiled module)... I'm sure there's some even easier way to add deps just to Gradle (`buildscript { ... }`?)

Reply:
Coz currently `jsoup` will be packaged into plugin zip itself, see `zipinfo -1 build/distributions/<current-version>.zip | sort`